Formula/o/odin.rb

```diff
@@ -83,6 +83,34 @@ def install
       end
     end
 
+    if OS.linux?
```

Comment:
Nothing in this block seems specific to Linux, except the naming of the shared library file (.so). Why not make only that an `if OS.mac? ... else ...` block?